### PR TITLE
aws endpoint deployment

### PR DESCRIPTION
**Pull request by aphawkins1**
…to create container to deploy

### CODE REVIEW

File: 007-AWS-Endpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+   "id": "949c47a6",
+   "metadata": {},
+   "source": [
+    "### Deploy the working service to AWS Serverless Lambda:\n",
+    "\n",
+    "Use the sam deployment command that guides you through the process:\n",
+    "\n",
+    "```sam deploy --guided```\n",
+    "\n",
+    "### Test the AWS endpoint using postmates:"
```
Comment (Andrewwango) [inline 007-AWS-Endpoint.ipynb:99]: `postman`

File: 007-AWS-Endpoint.ipynb [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+      "[1.67, 0.56, 0.72, 2.07, -0.35, 0.05, 0.54, 3.1, 2.49, 0.7, 0.28, 0.43, -0.12, 0.94, -0.26, -0.23, 0.32, -0.36, -0.52, 4.36, -0.74, 0.6, 0.14, 1.12, -0.37]\n",
```
Comment (Andrewwango) [inline 007-AWS-Endpoint.ipynb:171]: This is a bit messy - it's hard to tell which sentence corresponds to informal and formal, I'm not manually counting sentences to find the informal ones. Maybe in the UI include some logic to present it like
Informal: ......
Formal: .......
....
.....
Informal:......
.....
etc.

Or perhaps colour the sentences by informal (red) and formal (blue)?

File: formality-api/app/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FROM public.ecr.aws/lambda/python:3.9
+
+COPY app.py ./
+COPY requirements.txt ./
+COPY api ./api/
+COPY models/* ./models/
+
+RUN python3.9 -m pip install -r requirements.txt -t .
+RUN mkdir -p /var/lang/nltk_data
+RUN python3.9 -c "import nltk; a = [nltk.download(_, download_dir='/var/lang/nltk_data') for _ in ['punkt']]"
```
Comment (Andrewwango) [inline formality-api/app/Dockerfile:10]: Don't use a list comprehension for a list of size one. Just call the function

File: formality-api/app/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+FROM public.ecr.aws/lambda/python:3.9
```
Comment (Andrewwango) [inline formality-api/app/Dockerfile:1]: You said "Select amazon/python3.8-base as the base image" in the notebook but here you choose 3.9 base image. Which did you use?

**Review by Andrewwango — CHANGES_REQUESTED** [review]
Good folder structure, and well done for deploying and getting the inference pipeline working. It's clear what's going on, and you have kept it simple.
Some small comments to 
a) make the returned info more useful
b) ensure consistency in your docs

**Review by Andrewwango — APPROVED** [review]
Well done! I like the switch to nltk downloader CLI instead of `python -c` :)

**Review by MihneaBocaKainos — APPROVED** [review]

**Pull request merged**